R/download_dataset.R

```diff
@@ -0,0 +1,170 @@
+#' Download data for publication
+#'
+#' ...
+#'
+#' @param connection A valid connection to the ETN database.
+#' @param animal_project_code (string) Animal project to download data from.
+#' @param directory (string) Path to local download directory.
+#' @param scientific_name (string) One or more scientific names to filter on.
+#'
+#' @return ...
```

Comment:
This should be surely be present. 
Something like:
A list of csv files:
1. `filename1.csv`
2. ...

Reply:
Agree